tests/unit-tests/test-class-sensei-analysis-course-list-table.php

```diff
@@ -71,46 +71,6 @@ public function testPrepareItems_DateStartedFilterSet_SetsMatchingItems() {
 		self::assertSame( $expected, $this->export_items( $table->items ) );
 	}
 
-	public function testPrepareItems_DefaultDateStartedFilterSet_SetsMatchingItems() {
```

Comment:
It looks like it would be better to keep the test, but update the expected value.

Reply:
Makes sense. Test updated in b3334f4bff633b4e512863273b0c6cb0e5e3c712.